### PR TITLE
Feat: Prepare update to v1.5

### PR DESCRIPTION
**Pull request by metricaez**

### CODE REVIEW

**Comment by metricaez** [comment]
⚠️ **Do not merge**, tests were disabled for the sake of debugging but `create_default_with_extensions` on `command` is failing due to the change of the `from_genesis` method, needs to be refactored to `builder` before merging.

**Review by Moliholy — DISMISSED** [review]

**Review by Moliholy — APPROVED** [review]

**Pull request merged**
